src/components/Toolbar/Toolbar.jsx

```diff
@@ -15,14 +20,47 @@ function Toolbar() {
   const [classes, setClasses] = useState(DEFAULT_CLASSES);
 
   /************************************
-   * Functions
+   * Hooks
    ************************************/
 
-  function handleToggle() {
+  /**
+   * Handles opening/closing of mobile navigation toolbar
+   * @returns {void}
+   * @callback
+   */
+  const handleToggle = useCallback(() => {
+    //if closing (open == true) then resetting classNames
     isOpen ? setClasses(DEFAULT_CLASSES) : generateOpenClasses();
     setIsOpen(!isOpen);
-  }
+  }, [isOpen]);
+
+  /**
+   * Callback used by resize eventListener to close mobile nav bar in desktop view
+   * @returns {void}
+   * @callback
+   */ 
+  const memoNavCleanUp = useCallback(() => {
+    //if mobile bar is open and screen width is greater than medium breakpoint (see _screens.scss)
+    return (isOpen && window.innerWidth > MEDIUM_BREAKPOINT) ? handleToggle() : null;
+  }, [isOpen, handleToggle]);
+
+  useEffect(() =>{
+    //adds event listener for window resizing
+    window.addEventListener('resize', memoNavCleanUp);
+    return () => {
+      //removes the event listener whenever component unmounted
+      window.removeEventListener('resize', memoNavCleanUp);
+    };
+  }, [memoNavCleanUp]);
+
+  /************************************
+   * Functions
+   ************************************/
 
+  /**
+   * Applys open rule to className to add open attributes to parts of navbar
```

Comment:
VS Code caught a spelling error. Applys should be Applies.

Reply:
:,(